client/src/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import './bootstrap.min.css'
```

Comment:
Bootstrap isn't being used in this project.
Remove bootstrap css.

Reply:
@maw1a  done